rules/msk_topic_config.go

```diff
@@ -147,3 +178,54 @@ func (r *MSKTopicConfigRule) reportMissingReplicationFactor(runner tflint.Runner
 	}
 	return nil
 }
+
+const (
+	compressionTypeKey = "compression.type"
+	compressionTypeVal = "zstd"
+)
+
+var compressionTypeFix = fmt.Sprintf(`"%s" = "%s"`, compressionTypeKey, compressionTypeVal)
+
+func (r *MSKTopicConfigRule) validateCompressionType(
+	runner tflint.Runner,
+	config *hclext.Attribute,
+	configPairMap map[string]hcl.KeyValuePair,
+) error {
+	ctPair, hasCt := configPairMap[compressionTypeKey]
+	if !hasCt {
+		err := runner.EmitIssueWithFix(
+			r,
+			fmt.Sprintf("missing %s: it must be equal to '%s'", compressionTypeKey, compressionTypeVal),
+			config.Range,
+			func(f tflint.Fixer) error {
+				return f.InsertTextAfter(config.Expr.StartRange(), "\n"+compressionTypeFix)
```

Comment:
Found this neat way of defining a missing attribute right at the start of the block. 
I'm gonna backtrack and use the same technique for the replication factor